geopandas/_vectorized.py

```diff
@@ -974,6 +975,24 @@ def normalize(data):
     return out
 
 
+def make_valid(data):
+    if compat.USE_PYGEOS:
+        return pygeos.make_valid(data)
+    elif not compat.SHAPELY_GE_18:
+        raise ImportError(
```

Comment:
```suggestion
        raise NotImplementedError(
```

Reply:
Hi, the function is written but i cannot test if it works (shapely 1.8 installed)

I tried creating a new conda venv:

conda create python=3.10 --name geopandas-dev-shapely_1_7

and then install geopandas and downgrading shapely:

conda install -c conda-forge  geopandas
conda install -c conda-forge shapely=1.7

but it doesn't work.
Any suggestion?